Makefile

```diff
@@ -1,16 +1,35 @@
-include .ci/common.mk
+SELF_DIR := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
+include $(SELF_DIR)/.ci/common.mk
 
 coverfile         := cover.out
 test              := .ci/test-cover.sh
 
-install: install-glide
-	( cd linters/badtime ; glide install -v)
-	( cd utilities/mockclean ; glide install -v)
+TARGETS :=             \
```

Comment:
nit: is it worth adding a section to the README on the steps required to add a new package? e.g. adding the target here

Reply:
sure thing, added `DEVELOPER.md` with notes